docs/getting-started.md

```diff
@@ -108,6 +108,8 @@ Start by installing and setting up [MSYS2](https://www.msys2.org/).
 !!! important
     After installing MSYS2, you'll need to open **MSYS2 MinGW 64-bit** to
     perform building. Otherwise, you may see build failures for cc.exe.
+    Please make sure to not have any whitesspace in your build path or it will cause build errors.
+    E.g: `C:\Users\User Name\xqemu\build.sh` will not work.
```

Comment:
This is in the wrong location? I believe the MSYS2 docs warn about whitespaces in MSYS path.
We should probably warn about XQEMU spaces just before telling people to "Clone the repo" (which is entirely independent of MSYS installation - which most developers probably already have).